MergePDF/Form1.cs

```diff
@@ -92,14 +107,8 @@ private void btnMergePdf_Click(object sender, EventArgs e)
                 return;
             }
 
-            var pdfs = panel1.Controls.Cast<object>()
-             .Where(x => x is PdfSelectorControl)
-             .Select(x => x as PdfSelectorControl)
-             .OrderBy(x => x.Index)
-             .Where(x => string.IsNullOrWhiteSpace(x.FileName) == false)
-             .Select(x => x.FileName);
-
-            if (pdfs == null || pdfs.Count() < 2)
+            var pdfs = GetPdfs();
```

Comment:
If the type of `pdfs` is not known (i.e. not using a constructor), please do not use `var`.